adserver/api/views.py

```diff
@@ -296,6 +296,7 @@ def decision(self, request, data):
                 keywords=keywords,
                 campaign_types=serializer.validated_data.get("campaign_types"),
                 url=url,
+                placement_index=serializer.validated_data.get("index"),
```

Comment:
Should we call the incoming variable `placement_index` as well to keep it consistent?

Reply:
https://github.com/readthedocs/ethical-ad-client/pull/171